src/style.css

```diff
@@ -1 +1,12 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin: 10px 0;
+}
+
+.field-group {
+  margin: 20px 0;
```

Comment:
you should have added only bottom margins so that there is no top offset
```suggestion
  margin-bottom: 20px;
```
![Screenshot 2022-11-21 at 13 22 32](https://user-images.githubusercontent.com/75496546/203038260-36625c8c-3ffc-4f45-8ff9-cbd48d516b6f.png)